documentation/API-onboarding.md

```diff
@@ -1,81 +1,237 @@
-# API onboarding and management in CAMARA
+# API Onboarding and Lifecyle-Management in CAMARA
 
 ## Introduction
 
-This document aims at describing a clear, consistent and transparent process for the onboarding (submission) and management (maintenance) of API proposals and APIs in the CAMARA Project.
+_This document reflects current API onboarding process. Updated version including separated API and repository onboarding and lifecycle management will be included soon._
+
+
+This document describes clear, consistent, and transparent processes for the submission, evaluation, and onboarding of API proposals, and the management (maintenance and lifecycle evolution) of API Repositories in the CAMARA Project.
 
 The process pipeline is depicted in the figure below.
 
 ![API process pipeline](images/API_process_pipeline.png)
```

Comment:
I propose to delete these lines and with it the outdated picture even from this interim version of the document.
```suggestion
```